src/libtiled/wangset.cpp

```diff
@@ -593,7 +593,11 @@ void WangSet::removeWangTile(const WangTile &wangTile)
 QList<WangTile> WangSet::sortedWangTiles() const
 {
     QList<WangTile> wangTiles = mWangIdToWangTile.values();
+#if QT_VERSION < QT_VERSION_CHECK(5, 13, 0)
     qStableSort(wangTiles.begin(), wangTiles.end());
+#else
+    std::stable_sort(wangTiles.begin(), wangTiles.end());
+#endif
```

Comment:
Since the `std::stable_sort` also works with older Qt versions, there is no need for the Qt version check here.